ui/src/app/app.tsx

```diff
@@ -23,7 +23,8 @@ import {PKCEVerification} from './login/components/pkce-verify';
 services.viewPreferences.init();
 const bases = document.getElementsByTagName('base');
 const base = bases.length > 0 ? bases[0].getAttribute('href') || '/' : '/';
-export const history = createBrowserHistory({basename: base});
+const basePath = base.replace(/^https?:\/\/[^/]+/, '');
```

Comment:
See below, let's try to use some standard lib

Reply:
done

Used the JavaScript URL object to get the path.

const basePath = new URL(base, window.location.origin).pathname;